homeassistant/components/pglab/__init__.py

```diff
@@ -0,0 +1,72 @@
+"""PG LAB Electronics integration."""
+
+from __future__ import annotations
+
+from pypglab.mqtt import (
+    Client as PyPGLabMqttClient,
+    Sub_State as PyPGLabSubState,
+    Subcribe_CallBack as PyPGLaSubscribeCallBack,
+)
+
+from homeassistant.components import mqtt
+from homeassistant.components.mqtt import ReceiveMessage
+from homeassistant.components.mqtt.subscription import (
+    async_prepare_subscribe_topics,
+    async_subscribe_topics,
+    async_unsubscribe_topics,
+)
```

Comment:
PR #116150 has been merged, please change the import accordingly:
```suggestion
from homeassistant.components.mqtt import (
    ReceiveMessage,
    async_prepare_subscribe_topics,
    async_subscribe_topics,
    async_unsubscribe_topics,
)
```

Reply:
Ok... no problem